tools/ja/README.md

```diff
@@ -0,0 +1,34 @@
+# Proofreading tool for Japanese translation
+
+This is a proofreading tool for Japanese translation.
+We can check our documants following the configuration file: [`redpen-conf.xml`](redpen-conf.xml).
+For example, checking notations and synonyms with [`terminologies.txt`](terminologies.txt).
+
+## Requirements
+
+- [RedPen](http://redpen.cc/)
+  - We use RedPen to check markdown files
+- [Jupyter](https://jupyter.org/)
+  - We use Jupyter to convert notebooks to markdowns
+
+## Basic usage
+
+You can run the proofreading tool as below:
+
+```shell script
+./tools/ja/bin/run.sh <target directory or file>
+```
+
+For example:
+
+```shell script
+./tools/ja/bin/proofreading.sh site/ja/guide
```

Comment:
Tested this locally and seems to work ... I used homebrew to install the redpen package, probably out of scope to provide detailed install instructions, but maybe mention the Java dependency---speaking of Java, will this run in the GitHub Actions VM?

There was a log of noise at the command-line when running this, is that usual? Any way to filter out all these deprecation warnings?